ideascube/library/views.py

```diff
@@ -134,7 +134,8 @@ class SpecimenCreate(CreateView):
     def get_initial(self):
         book = get_object_or_404(Book, pk=self.kwargs['book_pk'])
 
-        return {'book': book, 'location': get_server_name()}
+        return {
+            'book': book, 'location': Configuration('server', 'site-name')}
```

Comment:
Yes, I disagree to change the name of a API to fulfill caller coding style.

User can still use : 
`from ideascube.configuration.models import Configuration as Cfg`